cmd/cassandra-reader/main.go

```diff
@@ -75,7 +78,9 @@ type config struct {
 	serverKey         string
 	jaegerURL         string
 	thingsAuthURL     string
+	usersAuthURL      string
```

Comment:
How do we set this, since there is no env var added? Also, when you load config, this field should be populated. Please check this for all the other readers.